### PR TITLE
simplify more forms

### PR DESCRIPTION
**Pull request by grosser**
@sandlerr @irwaters @jonmoter 

... I think I got everything now ... let me know if you find any non-input forms :)

### CODE REVIEW

File: plugins/kubernetes/app/views/samson_kubernetes/_build_new.erb [hunk 1]
```diff
@@ -1,6 +1 @@
-<div class="form-group">
-  <%= form.label :kubernetes_job, "Use Remote Kubernetes Job?", class: "col-lg-2 control-label" %>
-  <div class="col-lg-4">
-    <%= form.check_box :kubernetes_job %>
-  </div>
-</div>
+<%= form.input :kubernetes_job, as: :check_box, label: 'Use Remote Kubernetes Job?' %>
```
Comment (sandlerr) [inline plugins/kubernetes/app/views/samson_kubernetes/_build_new.erb]: does this still need to be a partial?

File: app/views/jobs/new.html.erb [hunk 2]
```diff
@@ -1,20 +1,11 @@
 <h1>Execute job on <%= @project.name %></h1>
 
 <%= form_for [@project, @job], html: { class: "form-horizontal" } do |form| %>
-  <fieldset>
-    <%= render 'shared/errors', object: @job %>
-  </fieldset>
```
Comment (sandlerr) [inline app/views/jobs/new.html.erb]: does losing the fieldsets affect the styling?

File: app/views/builds/new.html.erb [hunk 3]
```diff
@@ -2,53 +2,31 @@
...
-        <div id="scrollable-dropdown-menu" class="col-lg-4">
-          <%= form.text_field :git_ref, class: "form-control", autofocus: true, placeholder: "e.g. v2.1.43, master, fa0b4671", data: { prefetch_url: project_references_path(@project, format: "json") } %>
-        </div>
+    <%= form.input :git_ref, label: 'Git Reference' do %>
+      <div id="scrollable-dropdown-menu">
+        <%= form.text_field :git_ref, class: "form-control", autofocus: true, placeholder: "e.g. v2.1.43, master, fa0b4671", data: { prefetch_url: project_references_path(@project, format: "json") } %>
       </div>
+    <% end %>
 
-      <div>
```
Comment (grosser) [inline app/views/builds/new.html.erb]: yes, this was the form-group wrapper which is now included in `form.input`

Comment (grosser) [inline app/views/builds/new.html.erb]: ah nvm ... I'll check

File: app/views/jobs/new.html.erb (hunk 2 above)
Comment (grosser) [inline app/views/jobs/new.html.erb]: just a little bit of padding, barely noticeable

File: plugins/kubernetes/app/views/samson_kubernetes/_build_new.erb (hunk 1 above)
Comment (grosser) [inline plugins/kubernetes/app/views/samson_kubernetes/_build_new.erb]: yes, plugin views work by giving a partial to render ... just passing a block is not supported ... also keeps it consistent

**Pull request merged**